packages/nextra-theme/src/index.tsx

```diff
@@ -30,10 +28,118 @@ import {
 } from '@/components'
 import { DocumentContext, MDXLayoutNav, MDXLayoutOutline, MDXLayoutPagination, NavContext } from '@/layout'
 
+const MDXWrapper: NextraMDXContent = ({ children, toc }) => {
+  const { activePath } = useContext(NavContext)!
+  const { frontMatter, timestamp, readingTime } = useContext(DocumentContext)!
+  const { locale } = useRouter()
+  const lastUpdated = timestamp ? new Date(timestamp) : null
+
+  return (
+    <>
```

Comment:
Looks like this wrapper fragment is not needed.

Reply:
nope, we have 2 childrens 
<img width="444" alt="image" src="https://github.com/user-attachments/assets/77149017-4270-4404-a005-ff6efab170d3">

Reply:
🤦‍♂️

Thanks!